libraries/splunk_password.rb

```diff
@@ -0,0 +1,99 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: splunk_password.rb
+
+require 'base64'
+
+# Module contains different functions to encrypt and decrypt splunk passwords
+module CernerSplunk
+  # RC4 implementation taken from https://github.com/caiges/Ruby-RC4
```

Comment:
I prefer to use the class itself over installing the gem so that we can avoid the pain of bringing in an external dependency and managing it. 
I'm not sure about what needs to be done to conform to the MIT license standards. I'm looking into it now.